### PR TITLE
Update .k8s-helpers by fixing kpfp

### PR DESCRIPTION
**Pull request by suryapandian**
kpfp was returning with 
`No ports for  container in secret-pod-7c985b49dd-5jz7d`

This PR fixes the function

### CODE REVIEW

File: .k8s-helpers [hunk 1]
```diff
@@ -174,12 +174,12 @@ kpfp() {
...
   if [[ "$(jq '.spec.containers | length' <<< "${data}")" = 1 ]]; then
-    container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
+    local -r container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
   else
-    container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
+    local -r container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
   fi
   datajson="$(jq ".spec.containers[] | select(.name==\"${container}\")" <<< "${data}")"
-  if [[ "$(jq ".ports" <<< "${data}")" == "null" ]]; then
+  if [[ "$(jq ".ports" <<< "${datajson}")" == "null" ]]; then
```
Comment (suryapandian) [inline .k8s-helpers:182]: i think this is the main bit, we need to get the port from `datajson` and not `data`
not sure how this worked with `data` 🤔

File: .k8s-helpers [hunk 2]
```diff
@@ -174,12 +174,12 @@ kpfp() {
   local -r pod="${1:-$(kubectl get pod --no-headers | choose 0 | fzf)}"
   local -r data="$(kubectl get pod "${pod}" -ojson)"
   if [[ "$(jq '.spec.containers | length' <<< "${data}")" = 1 ]]; then
-    container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
+    local -r container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
   else
-    container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
+    local -r container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
```
Comment (suryapandian) [inline .k8s-helpers:179]: this is necessary too, coz in the lines below we are referring `$container` and it comes as empty without using `local -r`

**Review by clux — APPROVED** [review]
thanks!

File: .k8s-helpers (hunk 1 above)
Comment (suryapandian) [reply .k8s-helpers:182]: for posterity, we later discussed that this function was not used that much coz we all used `kpfs`

**Pull request merged**